Lib/pdb.py

```diff
@@ -3031,11 +3128,20 @@ def cmdloop(self):
                 self.process_payload(payload)
 
     def send_interrupt(self):
-        print(
-            "\n*** Program will stop at the next bytecode instruction."
-            " (Use 'cont' to resume)."
-        )
-        sys.remote_exec(self.pid, self.interrupt_script)
+        if hasattr(signal, "pthread_kill"):
```

Comment:
Now that we don't even use `pthread_kill`, I think we could have better options to detect platform. I think the most commonly used is `sys.platform == "win32"`.

Reply:
What we want here isn't really Windows vs not-Windows, it's Unix vs not-Unix.

For now `sys.platform == "win32"` is good enough, but if we ever got `sys.remote_exec()` and remote PDB ported to WASI or something weird like that, that guard might need updating.

I'll update it to `sys.platform == "win32"`, though, and leave that problem for another day.